.github/workflows/cd.yaml

```diff
@@ -0,0 +1,60 @@
+name: cd
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
```

Comment:
This is temporary. I'll remove it before merging. This is just to test this workflow on this PR.